### ISSUE TITLE
Unit tests for several API functions are failing with 400 due to invalid query arguments

### ISSUE THREAD

**Issue by akhtariev** [dhiaayachi/go-newton-co#4]
For example,

Running the `TestActions` unit test returns 400 with the body of:
```
"{\"limit\":[\"Provided limit invalid.\"],\"start_date\":[\"Provided start and end date invalid.\"]}"
```

A side note: Newton probably added input validation after this API was written 

The change should require modifying unit tests to provide valid inputs for all endpoints

**A/C**
- all unit tests pass
- API modified where needed

**Comment by akhtariev** [comment]
I will work on this in the next couple of days

**Comment by akhtariev** [comment]
was fixed in #5
